pages/architecture.md

```diff
@@ -4,53 +4,50 @@ weight: 2
 ---
 
 
-These are the components that make up a fully fledged tracing system.
-
-![Architecture overview]({{ site.github.url }}/public/img/architecture-0.png)
 
 Instrumented libraries
 ----------------------
 
 Tracing information is collected on each host using the instrumented libraries
 and sent to Zipkin. When the host makes a request to another service, it passes
-a few tracing identifers along with the request so we can later tie the data
+a few tracing identifiers along with the request so we can later tie the data
 together.
 
 ![Instrumentation architecture]({{ site.github.url }}/public/img/architecture-1.png)
 
 To see if an instrumentation library already exists for your platform, see the
-list of [existing instrumentations]({{ site.github.url
-}}/pages/existing_instrumentations).
+list of [existing instrumentations]({{ site.github.url}}/pages/existing_instrumentations).
+
+![Architecture overview]({{ site.github.url }}/public/img/architecture-0.png)
 
 Transport
 ---------
 
-Spans must be transported from the services being traced to Zipkin collectors.
-There are two primary transports, Scribe and Kafka. Scribe is deprecated.
+Spans sent by the instrumented library must be transported from the services being traced to Zipkin collectors.
+There are two primary transports: Scribe and Kafka. Scribe is shown in the figure. See [Span Receivers]({{ site.github.url }}/pages/span_receivers) for more information.
+
+There are 4 components that make up Zipkin:
+* collector
+* storage
+* search
+* web UI
 
-Zipkin Collector
-----------------
+### Zipkin Collector
```

Comment:
Probably depends on how we phrase things, but by default I expect this to cause confusion. Consider: "trace data is sent from instrumented applications through one of various transports, and is received by the appropriate transport component in Zipkin".

I think in the context of Zipkin source code it's fine, but in the context of the architecture description we may want another word.